### PR TITLE
fix: electron locator bug

### PR DESCRIPTION
**Pull request by chase-itay**
bug the await is on nothing because the brackets [] have precedence

### CODE REVIEW

File: src/electron-locator.ts [hunk 1]
```diff
@@ -20,7 +20,8 @@ async function locateModuleByRequire(): Promise<string | null> {
 
 export async function locateElectronModule(projectRootPath?: string): Promise<string | null> {
   for (const moduleName of electronModuleNames) {
-    const electronPath = await searchForModule(process.cwd(), moduleName, projectRootPath)[0];
+    let paths = await searchForModule(process.cwd(), moduleName, projectRootPath);
+    const electronPath = paths[0];
```
Comment (malept) [inline src/electron-locator.ts:24]:
```suggestion
    const electronPath = (await searchForModule(process.cwd(), moduleName, projectRootPath))[0];
```

**Review by malept — CHANGES_REQUESTED** [review]
Thanks for spotting this! Can you add a regression test?

**Comment by malept** [comment]
@chase-itay are you interested in finishing this PR so it can be merged?

**Review by danielehrhardt — APPROVED** [review]

**Comment by malept** [comment]
I'm closing this PR for now, but if you would like to revive it, please rebase, add a regression test, and you should be able to reopen it.

**Pull request closed without merging**